Remove DeviceView from ListDevicesByUser

avatus opened the pull request:
Originally, we included the `View` field in order to add the Owner property to safeguard the returned details of a device ensure the owner matched the caller. However, this ended up not being conditionally used (we would never _not_ use it) so we can remove it from the request.

This proto hasn't been used anywhere yet so I think its ok to remove the field instead of marking as reserved? you let me know @codingllama 

edit: i guess the linter is going to make us keep this field

codingllama reviewed (commented):
No complaints about removing views from my part, but the linter is indeed going to ask you to reserve the field name/tag.

avatus committed:
Remove DeviceView from ListDevicesByUser

Originally, we included the `View` field in order to add the Owner
property to safeguard the returned details of a device ensure the owner
matched the caller. However, this ended up not being conditionally used
(we would never _not_ use it) so we can remove it from the request.
